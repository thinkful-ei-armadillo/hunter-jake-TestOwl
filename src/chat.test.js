import React from 'react';
import ReactDOM from 'react-dom';
import Chat from './Chat';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Chat />, div);
  ReactDOM.unmountComponentAtNode(div);
});

describe('<Chat />', () => {
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Chat />)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });
})