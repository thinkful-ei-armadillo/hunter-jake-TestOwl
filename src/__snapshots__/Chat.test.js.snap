// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chat /> renders the UI as expected 1`] = `
<div
  className="chatLog"
>
  <div>
    <ul>
      <li>
        <p>
          Axe Kubicka
        </p>
        <p>
          5
          :
          48
        </p>
      </li>
      <p>
        Hello world
      </p>
    </ul>
    <ul>
      <li>
        <p>
          Caty Flucker
        </p>
        <p>
          5
          :
          48
        </p>
      </li>
      <p>
        gave a thumbs up
      </p>
    </ul>
    <ul>
      <li>
        <p>
          Frank Runciman
        </p>
        <p>
          5
          :
          48
        </p>
      </li>
      <p>
        gave a thumbs down
      </p>
    </ul>
    <ul>
      <li>
        <p>
          Koren Templeton
        </p>
        <p>
          5
          :
          49
        </p>
      </li>
      <p>
        raised hand
      </p>
    </ul>
    <ul>
      <li>
        <p>
          Koren Templeton
        </p>
        <p>
          5
          :
          49
        </p>
      </li>
      <p>
        clapped
      </p>
    </ul>
    <ul>
      <li>
        <p>
          Koren Templeton
        </p>
        <p>
          5
          :
          49
        </p>
      </li>
      <p>
        joined
      </p>
    </ul>
    <ul>
      <li>
        <p>
          Ashla Attwool
        </p>
        <p>
          5
          :
          50
        </p>
      </li>
      <p>
        left
      </p>
    </ul>
    <ul>
      <li>
        <p>
          Axe Kubicka
        </p>
        <p>
          5
          :
          51
        </p>
      </li>
      <p>
        joined the stage
      </p>
    </ul>
    <ul>
      <li>
        <p>
          Axe Kubicka
        </p>
        <p>
          5
          :
          52
        </p>
      </li>
      <p>
        left the stage
      </p>
    </ul>
     
  </div>
</div>
`;
